invokeai/backend/stable_diffusion/diffusers_pipeline.py

```diff
@@ -207,8 +202,8 @@ class ControlNetData:
 
 @dataclass
 class ConditioningData:
-    unconditioned_embeddings: torch.Tensor
-    text_embeddings: torch.Tensor
+    unconditioned_embeddings: Any # TODO: type
+    text_embeddings: Any # TODO: type
```

Comment:
Are these not tensors anymore?

Reply:
Yes, now it's conditionings object with all info about this conditionings.